doc/release_notes.rst

```diff
@@ -7,7 +7,10 @@ Upcoming Release
 
 .. warning:: The features listed below are not released yet, but will be part of the next release! To use the features already you have to install the ``master`` branch, e.g. ``pip install git+https://github.com/pypsa/pypsa#egg=pypsa``.
 
-* new features
+**New Features**
+
+* PyPSA can import pandapower networks. Now, some issues regarding the transformer component and indexing are fixed. Shunts can also be imported. [`#332 <https://github.com/PyPSA/PyPSA/pull/332>`_]
```

Comment:
PyPSA could import from pandapower for earlier versions. Maybe rephrase to: "restore import of pandapower networks".